ants/ant.cpp

```diff
@@ -295,19 +295,17 @@ std::vector<ant> move(std::vector <ant> ants, grid landscape, coord spawn,
 
         }
 
-        std::sort(std::begin(killlist), std::end(killlist),
-                  [](int &dum1, int &dum2)
-                     {return dum1 > dum2;});
+        std::reverse(std::begin(killlist), std::end(killlist));
 
         std::cout << "size: " << killlist.size() << '\n';
         for (size_t k = 0; k < killlist.size(); k++){
             ants.erase(ants.begin() + killlist[k]);
         }
-        killlist = {};
+        killlist.clear();
```

Comment:
This reuses the existing buffer (your version has the potential to allocate on every iteration).